runtime/compilers/rillv1/parse_model.go

```diff
@@ -106,6 +112,44 @@ func (p *Parser) parseModel(node *Node) error {
 		inputProps["sql"] = sql
 	}
 
+	// special handling to mark model as updated when local file changes
+	if inputConnector == "local_file" {
+		c, ok := inputProps["invalidate_on_change"].(bool)
```

Comment:
If not explicitly set to `false`, should we let it default to `true`? I guess that's the behavior most folks would expect by default.

Reply:
I was just worried about somebody adding a big file in local modelling. But yeah `true` by default can also be set.